server/storages/postgres.go

```diff
@@ -260,3 +260,7 @@ func (p *Postgres) Close() (multiErr error) {
 
 	return
 }
+
+func (p *Postgres) TestBatchProcessing(testName string, events []map[string]interface{}) error {
```

Comment:
same, like ClickHouse

Reply:
Refactoring made `batch` testing available for `Postrgres` storage type.